site/static/compare/script.js

```diff
@@ -433,6 +387,86 @@ app.component('test-cases-table', {
 </div>
 `
 });
+
+const SummaryPercentValue = {
+    props: {
+        value: Number,
+        padWidth: {
+            type: Number,
+            default: null
+        }
+    },
+    template: `
+<span><span v-html="padSpaces" />{{ formattedValue }}%</span>
+`,
+    computed: {
+        formattedValue() {
+            return `${this.signIfPositive(this.value)}${this.value.toFixed(2)}`;
+        },
+        padSpaces() {
+            let value = this.formattedValue;
+            if (value.length < this.padWidth) {
+                return "&nbsp;".repeat(this.padWidth - value.length);
+            }
+            return "";
+        }
+    }
+};
+const SummaryRange = {
+    props: {
+        range: Array,
+    },
+    template: `
+<div v-if="range.length > 0">
+  [<SummaryPercentValue :value="range[0]" :padWidth="6" />, <SummaryPercentValue :value="range[1]" :padWidth="6" />]
+</div>
+<div v-else>-</div>
+`, components: {SummaryPercentValue}
+};
+const SummaryCount = {
+    props: {
+        cases: Number,
+        benchmarks: Number
+    },
+    template: `
+<span :title="cases + ' test case(s), ' + benchmarks + ' unique benchmark(s)'">{{ cases }} ({{ benchmarks }})</span>
+`
+};
+
+app.component('summary-table', {
+    props: ['cases'],
+    template: `
+<table class="summary-table">
+    <thead>
+      <th><!-- icon --></th>
+      <th>Range</th>
+      <th>Mean</th>
+      <th>Count</th>
+    </thead>
+    <tbody>
+        <tr class="positive">
+            <td title="Regresions">❌</td>
```

Comment:
```suggestion
            <td title="Regressions">❌</td>
```

Reply:
Oops :) Thanks. Will fix in https://github.com/rust-lang/rustc-perf/pull/1410 if it gets merged, otherwise I'll send a separate PR.